src/blocks/gallery-carousel/edit.js

```diff
@@ -131,277 +117,261 @@ class GalleryCarouselEdit extends Component {
-											<GalleryImage
-												url={ img.url }
-												alt={ img.alt }
-												id={ img.id }
-												gutter={ gutter }
-												gutterMobile={ gutterMobile }
-												marginRight={ true }
-												marginLeft={ true }
-												isSelected={ isSelected && this.state.selectedImage === index }
-												onRemove={ this.onRemoveImage( index ) }
-												onSelect={ this.onSelectImage( index ) }
-												setAttributes={ ( attrs ) => this.setImageAttributes( index, attrs ) }
-												caption={ img.caption }
-												aria-label={ ariaLabel }
-												supportsCaption={ false }
-												supportsMoving={ false }
-												imageIndex={ index }
-												replaceImage={ this.replaceImage }
-											/>
-										</div>
-									);
-								} ) }
-							</Flickity>
-						</div>
-					</div>
-				</ResizableBox>
-				{ thumbnails &&
-					<div className={ className }>
-						<div
-							className={ innerClasses }
-							style={ navStyles }
+	);
+
+	const flickityClasses = classnames(
+		'has-carousel',
+		`has-carousel-${ gridSize }`, {
+			'has-aligned-cells': alignCells,
+			[ `has-margin-bottom-${ gutter }` ]: thumbnails && gutter > 0,
+			[ `has-margin-bottom-mobile-${ gutterMobile }` ]: thumbnails && gutterMobile > 0,
+			[ navForClass ]: thumbnails,
+		}
+	);
+
+	const navClasses = classnames(
+		'carousel-nav', {
+			[ `has-margin-top-${ gutter }` ]: gutter > 0,
+			[ `has-margin-top-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+			[ `has-negative-margin-left-${ gutter }` ]: gutter > 0,
+			[ `has-negative-margin-left-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+			[ `has-negative-margin-right-${ gutter }` ]: gutter > 0,
+			[ `has-negative-margin-right-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+		}
+	);
+
+	const flickityOptions = {
+		draggable: false,
+		pageDots: true,
+		prevNextButtons: true,
+		wrapAround: true,
+		autoPlay: false,
+		cellAlign: alignCells ? 'left' : 'center',
+		arrowShape: {
+			x0: 10,
+			x1: 60, y1: 50,
+			x2: 65, y2: 45,
+			x3: 20,
+		},
+		responsiveHeight,
+		thumbnails,
+	};
+
+	const navOptions = {
+		asNavFor: `.${ navForClass }`,
+		draggable: false,
+		pageDots: true,
+		prevNextButtons: false,
+		wrapAround: true,
+		autoPlay: false,
+		thumbnails: false,
+		cellAlign: 'left',
+	};
+
+	const navStyles = {
+		marginTop: gutter > 0 && ! responsiveHeight ? ( gutter / 2 ) + 'px' : undefined,
+	};
+
+	const navFigureClasses = classnames(
+		'coblocks--figure', {
+			[ `has-margin-left-${ gutter }` ]: gutter > 0,
+			[ `has-margin-left-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+			[ `has-margin-right-${ gutter }` ]: gutter > 0,
+			[ `has-margin-right-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+		}
+	);
+
+	return (
+		<>
+			{ isSelected &&	<Controls { ...props } /> }
```

Comment:
not important but you could combine this conditional render into one condition:

```
{ isSelected && (
    <>
        <Controls {...props} />
        <Inspector {...props />
    </>
) }
```

Reply:
+1